src/agenda/index.js

```diff
@@ -27,6 +27,7 @@ export default class AgendaView extends Component {
   static displayName = 'Agenda';
 
   static propTypes = {
+    setRef: PropTypes.func,
```

Comment:
I don't see the use of this 'setRef' prop. Why not pass ref to the Agenda component and use it to invoke the closeCalendar() method?

Reply:
`setRef` used in `componentDidmount`. I pass `this` to setRef so parent class can access `closeCallendar` method of this class.

Reply:
Again, why not use the ref? pass `ref={calendarRef => this.agendaRef = calendarRef}` to Agenda component and use it the same way you do with `setRef`.

Reply:
because ref not working.

Reply:
my bad. to be honest, I forgot why I put setRef instead of ref. 😅
But, I think with setRef we can set which methods that we want to pass to the parent instead of pass all methods (but, I haven't added this to the code also).

Reply:
And that's one of the reasons I don't like the `setRef`...